rootlesscontainers.proto

```diff
@@ -19,6 +19,8 @@ syntax = "proto3";
 // If you want to extend the resource definition, please open a PR.
 package rootlesscontainers;
 
+option go_package = "github.com/rootless-containers/proto/go-proto";
```

Comment:
Could you update the go file too?
Then LGTM

Reply:
👍